media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="dav1d is a new AV1 cross-platform decoder, open-source, \
+and focused on speed and correctness. It supports all features from AV1, \
+including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="3.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	dav1d$secondaryArchSuffix = $portVersion
+	cmd:dav1d
+	lib:libdav1d = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	dav1d${secondaryArchSuffix}_devel = $portVersion
+	devel:libdav1d = $libVersionCompat
+	"
+REQUIRES_devel="
+	dav1d$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:meson >= 0.47
+	cmd:nasm >= 2.13.02
```

Comment:
nasm bump is at 2.14.02
```diff
- 	cmd:nasm >= 2.13.02
+ 	cmd:nasm >= 2.14.02
```

Reply:
However, [dav1d's documentation](https://code.videolan.org/videolan/dav1d#compile) says that it requires 2.13.02 only.
So if some guy managed to compile 2.13.02, he can still use this for building dav1d.

Reply:
I doubt that we'll go back to 2.13.02 if we have 2.14.02, so there won't be anything providing 2.13.02 (doesn't error out, but it will check the current version we are having)

Reply:
I'd say this is a non-issue :P